src/js/core.js

```diff
@@ -873,6 +873,16 @@
             }
         },
 
+        clearSelection: function (toEnd) {
```

Comment:
Any reason not to call this `collapseSelection()` (to use an already existing, well-defined term)?